sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -476,6 +476,23 @@ object SQLConf {
     .intConf
     .createWithDefault(10000)
 
+  val VECTORIZED_HUGE_VECTOR_RESERVE_RATIO =
+    buildConf("spark.sql.inMemoryColumnarStorage.hugeVectorReserveRatio")
+      .doc("spark will reserve requiredCapacity * this ratio memory next time")
```

Comment:
I think it worth mention here this ratio is only effective when `VECTORIZED_HUGE_VECTOR_THRESHOLD` is enabled and required mem larger that threshold

Reply:
Done